### ISSUE TITLE
Don't enable the systemd mullvad-daemon service after install

### ISSUE THREAD

**Issue by rwanyoike** [TimoKramer/mullvad-vpn-cli#1]
This package enables and starts itself on install and upgrade. This is quite unusual for Arch packages and violates the principle of least surprise. Please consider removing that from the install script. 

Eg: When this installed on a remote/ssh instance, you're immediately locked out of the system, after the daemon sets up the internet kill-switch, blocking all traffic outside the mullvad vpn/local network.

**Comment by TimoKramer** [comment]
right, I was never considering the case someone wants to use mullvad-client on a remote system. What do you think it should do? Should the enable be left out and maybe only mentioned as a notice at the finish of install?

**Comment by rwanyoike** [comment]
Thanks for this package! 

As with most other packages I've experienced on Arch, I think it should just install mullvad, only that. Leave it up to the user to know what to do with it (use systemd/call it directly/poke around etc...)

**Comment by TimoKramer** [comment]
I am glad you find it useful! I am thinking about not enabling it... Since people relying on it to run maybe for security/privacy it should be there after a restart, I think. Maybe it is ok to enable it but leave away the `--now`? And echoing that people need to start it with the command needed.
